src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adderfunc = (a) => {
+    if (a == null) {
```

Comment:
Instead of checking for null explicitly, it's more common in JavaScript to check for a falsy value to cover `undefined`, which is what you get from calling a function without arguments. You can use `if (!a)` instead.